mParticle-ComScore.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
     s.ios.deployment_target = "8.0"
     s.ios.source_files      = 'mParticle-ComScore/*.{h,m,mm}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 6.12.0'
```

Comment:
Not a big real really, just to set the anchor version of the core SDK to _6.12.3_

```
s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 6.12.3'
```